charts/kor/values.yaml

```diff
@@ -14,6 +14,45 @@ cronJob:
   successfulJobsHistoryLimit: 3
   failedJobsHistoryLimit: 2
 
+prometheusExporter:
+  enabled: true
+  name: kor-exporter
+  # time in minutes, default is 10 minutes
+  exporterInterval: ""
+  deployment:
+    image:
+      repository: yonahdissen/kor
+      tag: latest
+    command: kor exporter
```

Comment:
Wouldn't it be better if this command specified the destination where it's exporting to? I mean maybe in the future some other feature could try to export somewhere else other then to Prometheus.
What do you think of having something like:

`kor exporter --prometheus`
or
 `kor prom-exporter`